src/App.tsx

```diff
@@ -0,0 +1,184 @@
+    const example_code = await getExample(value)
+    setInputs(await example_code[0])
+    setCode(await example_code[1])    
+
+    // reset the output
+    setOutput(emptyOutput)
+  }
+
+  return (
+    <>
+      <AppBar position="static">
+        <Toolbar variant="dense">
+          <Typography variant="h6" color="inherit" component="div">
+            Playground for Miden Examples in Miden Assembly
+          </Typography>
+        </Toolbar>
+      </AppBar>
+      <Container maxWidth="xl">
+      <Box sx={{ display: 'flex', flexDirection: 'row', mx: 'auto', my: 4}}>
+          <FormControl variant="outlined" sx={{minWidth: 120}}>
+              <InputLabel id="select-example">
+                Example
+              </InputLabel>
+              <Select
+                value={example}
+                onChange={handleSelectChange}
+                label="Example"
+                labelId='select-example'
+              >
+                {examples.map((name) => (
+                  <MenuItem value={name} key={name}>{name}</MenuItem>
+                ))}
+              </Select>
+          </FormControl>
+          <FormControl variant="outlined" sx={{minWidth: 120, mx: 2}}>
+              <InputLabel id="select-outputs">
+                Outputs
+              </InputLabel>
+              <Select
+              value={numOfOutputs}
+              onChange={(e) => setNumOfOutputs(Number(e.target.value))}
+              label="Outputs"
+              labelId='select-outputs'
+            >
+              <MenuItem value={0} key={0}>0</MenuItem>
+              <MenuItem value={1} key={1}>1</MenuItem>
+              <MenuItem value={16} key={16}>16</MenuItem>
+            </Select>
+          </FormControl>
+          <Button
+            variant="contained"
+            color="primary"
+            onClick={() => {
+              init().then(() => {
+                try {
+                  let resp = program(code, inputs, numOfOutputs);
+                  console.log(resp);
+                  setOutput({
+                    init: true,
+                    success: true,
+                    text: resp.join(" "),
+                  });
+                } catch (error) {
+                  setOutput({
+                    init: true,
+                    success: false,
+                    text: "Error: Check the developer console for details.",
+                  });
+                }
+              });
+            }}
+            endIcon={<SendIcon />}
+          >
+            Execute
+          </Button>
+          {output.init ? (
+            <Alert
+              variant="outlined"
+              severity={output.success ? "success" : "error"}
+              sx={{mx: 2}}
+            >
+              <Typography color="inherit" component="div">
+                {output.text}
+              </Typography>
+            </Alert>
+          ) : (
+            <></>
+          )}
+        </Box>
```

Comment:
For the future: It would be good to pull this out into its own component and add it at both the top and the bottom of the screen. I think that would provide better UX (people wouldn't need to scroll back up after writing their code). Do you agree? If so, can you create an issue?

Reply:
Yes, I will create an issue